packages/date-picker/src/vaadin-infinite-scroller.js

```diff
@@ -315,43 +331,43 @@ class InfiniteScroller extends PolymerElement {
         const contentId = (InfiniteScroller._contentIndex = InfiniteScroller._contentIndex + 1 || 0);
         const slotName = `vaadin-infinite-scroller-item-content-${contentId}`;
 
-        const insertionPoint = document.createElement('slot');
-        insertionPoint.setAttribute('name', slotName);
-        insertionPoint._itemWrapper = itemWrapper;
-        buffer.appendChild(insertionPoint);
+        const slot = document.createElement('slot');
+        slot.setAttribute('name', slotName);
+        slot._itemWrapper = itemWrapper;
+        buffer.appendChild(slot);
 
         itemWrapper.setAttribute('slot', slotName);
         this.appendChild(itemWrapper);
 
-        setTimeout(() => {
-          // Only stamp the visible instances first
-          if (this._isVisible(itemWrapper, container)) {
-            this._ensureStampedInstance(itemWrapper);
-          }
-        }, 1); // Wait for first reset
```

Comment:
Using `setTimeout` here was added in https://github.com/vaadin/vaadin-date-picker/pull/268 to defer rendering of the scrollers until the overlay is opened. See https://github.com/vaadin/vaadin-date-picker/pull/268/commits/6709076f95ced32098f61357c6703bd7f517a623

Originally, the "lazy loading" was relying on animation events. However, later this logic was changed in https://github.com/vaadin/vaadin-date-picker/pull/297, see https://github.com/vaadin/vaadin-date-picker/commit/fff65dff7fb89a01590198c3674ac5d8e6b63e4b

After some debugging and testing, I don't see any reason for keeping `setTimeout` here. 
There is already `afterNextRender` used to render instances that aren't initially visible.